README.md

```diff
@@ -5,44 +5,39 @@
 ```bash
 git clone -b linked-examples https://github.com/willcrichton/rust
 cd rust
-./x.py --stage 1 build
-export CUSTOM_RUSTDOC=$(pwd)/build/x86_64-apple-darwin/stage1/bin/rustdoc
+./x.py build
+# replace `apple-darwin` with your current target as appropriate
+rustup toolchain link custom-rustdoc build/x86_64-apple-darwin/stage1
 cd ..
 git clone https://github.com/willcrichton/example-analyzer
-cd example_analyzer
-rustup toolchain install nightly --profile default --component rustc-dev
-rustup override set nightly
+cd example-analyzer
 cargo build
 ```
 
 ## Example
 
-
 ```bash
+# NOTE: the directory you run this from is important since the project uses
+# `rust-toolchain`
```

Comment:
This will also work from `doctest/`, but not in your home directory or something.